_posts/2021-01-27-#20833.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: pr
+date: 2021-01-27
+title: "Enable packages through testmempoolaccept"
+pr: 20833
+authors: [glozow]
+components: ["rpc/rest/zmq", "tests", "validation"]
+host: glozow
+status: upcoming
+commit: ebca80d
+---
+
+## Definitions
+
+- A **package** is a group of transactions that have some dependency
+  relationship, i.e., some transaction spends the outputs of another
+  transaction in the package.  We might call a package a "transaction chain" when
+  it has multiple generations of such dependencies.
```

Comment:
I'd omit this second sentence, since I think it probably makes things more confusing rather than clarifying:

- a package might not just be a single chain (i.e. each parent only has one child/each child only has one parent)
- we already have chains in Bitcoin :)

Reply:
Oopsie. Yeah, I guess it's more like "I sometimes say transaction chain"